Cargo.toml

```diff
@@ -103,7 +103,7 @@ wascap = { version = "0.11", default-features = false }
 wasi-common = { version = "11", default-features = false }
 wasmcloud-actor = { path = "./crates/actor" }
 wasmcloud-actor-derive = { path = "./crates/actor/derive" }
-wasmcloud-component-adapters = { path = "./crates/component-adapters" }
+wasmcloud-component-adapters = { git = "https://github.com/wasmCloud/wasmcloud-component-adapters" }
```

Comment:
should this be pinned to a hash?

Reply:
no, in fact we should be using versioned releases here, but I do not have the means to release at the moment